cypress/integration/apps/citas/agendas/planificacion_agendas.spec.js

```diff
@@ -0,0 +1,267 @@
+/// <reference types="Cypress" />
+
+const complete = (dto) => {
+    if (dto.fecha) {
+        cy.plexDatetime('label="Fecha"', dto.fecha);
+    }
+
+    if (dto.horaInicio) {
+        cy.plexDatetime('label="Inicio"', dto.horaInicio);
+    }
+
+    if (dto.horaFin) {
+        cy.plexDatetime('label="Fin"', dto.horaFin);
+    }
+
+    if (dto.cantidadTurnos) {
+        cy.plexInt('label="Cantidad de Turnos"', dto.cantidadTurnos);
+    }
+
+    if (dto.accesoDirectoDelDia) {
+        cy.plexInt('name="accesoDirectoDelDia"', dto.accesoDirectoDelDia);
+    }
+
+    if (dto.bloque) {
+        if (dto.bloque.horaInicio) {
+            cy.plexDatetime('label="Hora Inicio"', dto.bloque.horaInicio);
+        }
+
+        if (dto.bloque.horaFin) {
+            cy.plexDatetime('label="Hora Fin"', dto.bloque.horaFin);
+        }
+
+        if (dto.bloque.cantidadTurnos) {
+            cy.plexInt('label="Cantidad de Turnos"', dto.bloque.cantidadTurnos);
+        }
+
+        if (dto.bloque.accesoDirectoDelDia) {
+            cy.plexInt('name="accesoDirectoDelDia"', dto.bloque.accesoDirectoDelDia);
+        }
+    }
+}
+
+context('Planificacion Agendas', () => {
+    let token
+    before(() => {
+        cy.seed();
+        cy.viewport(1280, 720);
+        cy.login('38906735', 'asd').then(t => {
+            token = t;
+        });
+    })
+
+    beforeEach(() => {
+
+    });
+
+    it('crear agenda con un solo bloque', () => {
+        cy.goto('/citas/gestor_agendas', token);
+
+        cy.plexButton("Crear una nueva agenda").click();
+
+        cy.swal('cancel')
+
+        cy.server();
+
+        cy.route('POST', '**/api/modules/turnos/agenda**').as('create');
+
+        complete({
+            fecha: '03/10/2019',
```

Comment:
No dejar fechas hardcodeadas. 

```suggestion
            fecha: cy.today(),
```

Se puede granularizar más los test para cubrir distintas fechas. Anteriores y posteriores.

Reply:
Resuelto en 6b45037c9a172614c9885edb38cb96d6bf02646d